### ISSUE TITLE
Feature Request: react-router-redux v5.0

### ISSUE THREAD

**Issue by Vandivier** [davezuko/react-redux-starter-kit#1217]
react-router-redux v5.0 is now added to the [react-router repo](https://github.com/ReactTraining/react-router). That makes it the new standard for redux apps and this is the perfect boilerplate repo for it. Dependency on react-router v4.0.

**Comment by TheLordHighExecutioner** [comment]
I can rewrite the code in RR4 + RRR5 if the repo owners are interested.

**Comment by davezuko** [comment]
To be honest I'm not a huge fan of React Router 4.0 yet, so I'm wary of upgrading, but will consider it if I can get some heads down time with the new version.

**Comment by MidnightP** [comment]
Any updates on this yet?

**Comment by bman654** [comment]
I'm loving react-router 4, so much easier to create route-specific rendering in different areas of the page.  Though I'll admit it seems at odds with the way this repo implements per-route async module loading

**Comment by chin-phang** [comment]
Hi, I used react-router-starter-kit for production. I love it so I tried implement react-router 4 with react-router-redux 5 using create-react-app. [react-redux-quickstart](https://github.com/chin-phang/react-redux-quickstart)

**Comment by clearly** [comment]
+1...we are upgrading to v4 on our 2nd project based off of this repo. Is there a paypal link somewhere that I can contribute some $ to get this done quickly?

**Comment by amangeot** [comment]
Hello @bman654, sorry for asking a long time after you commented,

How do you deal with async redux reducers with react router 4?

**Comment by bman654** [comment]
@amangeot I don't use async module loading so I just removed all of that code as part of moving to router 4.

**Comment by amangeot** [comment]
Oh ok, thank you for the quick reply.

Did you also remove async components? If not, how do you deal with non-blocking transition from one route to another? To replace the behaviour that came from getComponent?

**Comment by bman654** [comment]
yes that is what I meant by not using async module loading - I load all of my components at the start of the app.  Check out [async-react-components](https://www.npmjs.com/package/async-react-component) for a way to load your components asynchronously with Router 4.

**Comment by kodermax** [comment]
when?

**Comment by tommyalvarez** [comment]
News on this?
